02-express-tutorial/04-express-app.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express')
+const path = require('path')
+const app = express()
+
+app.use(express.static('./public'))
+
+app.get('/', (req,res)=> {
+  res.sendFile(path.resolve(__dirname,'./navbar-app/index.html'))
+})
+
+app.get('*', (req,res)=> {
+  res.status(404).send('resource not found')
+})
+
+
+app.listen(5000, () => {
+  console.log('server is listening on port 5000...')
+})
```

Comment:
Extra challenge: Can you make it so that we can specify a different port with an environment variable, and also ensure that the `console.log` message updates accordingly?